lib/index.js

```diff
@@ -514,10 +514,13 @@ api.verify = async ({
 
   // TODO: could call `api.get()` here if not for the need for
   //  `record.account.id` at the end
-  // FIXME: this code is problematic if `account` is not set (likely being
-  // skipped when actor is null, but don't have test coverage when it isn't)
-  await brPermissionCheck(
-    actor, PERMISSIONS.ACCOUNT_ACCESS, {resource: [account]});
+  if(type !== 'nonce') {
+    if(!account) {
+      throw new Error('"account" must be given.');
```

Comment:
Is BedrockError too heavy weight here?  Maybe this conditional should just be an non-optional assert?